pkg/eks/update.go

```diff
@@ -162,6 +163,27 @@ func (c *ClusterProvider) UpdateClusterVersion(cfg *api.ClusterConfig) (*awseks.
 	return output.Update, nil
 }
 
+// UpdateClusterTags calls eks.TagResource and tags the cluster
+func (c *ClusterProvider) UpdateClusterTags(clusterConfig *api.ClusterConfig) error {
+	if err := c.RefreshClusterStatus(clusterConfig); err != nil {
+		return err
+	}
+	input := &awseks.TagResourceInput{
+		ResourceArn: c.Status.clusterInfo.cluster.Arn,
+		Tags:        utilsstrings.ToPointersMap(clusterConfig.Metadata.Tags),
```

Comment:
Here is the definition for this field in eks/api.go, it requires at at least 1 element 

```
     Tags map[string]*string `locationName:"tags" min:"1" type:"map" required:"true"`
```

Understand that you already have a length check for config.Metadata.Tags before calling UpdateClusterTags method, however this method is public, there might be the casse going the line, it will be called directly with empty tags (which eventually causes issue). 

What do think about utilsstring.NilPointersMapIfEmpty(utilsstrings.ToPointersMap(clusterConfig.Metadata.Tags)) ?

Reply:
True, but doesn't `required` mean `nil` would be a problem too?

Reply:
oh you are right. Grrr I should change to bigger Macbook (maybe 15 or 16 inch one), missed that part first time :)

Reply:
I will include the length check inside the update function